Frontend/src/common/components/LoginPage.tsx

```diff
@@ -57,7 +58,12 @@ const LoginPage = () => {
     e.preventDefault();
 
     signIn(login, password)
-      .then(() => {
+      .then(user => {
+        if (
+          process.env.REACT_APP_FOR === 'admin' &&
```

Comment:
Ja to bym wyciągnął gdzieś to process.env.REACT_APP_FOR === 'admin', bo to już drugie miejsce gdzie tego używamy, ale jako, że dopiero drugie, to nie naciskam jeszcze

Reply:
Jedyny sposób w jaki można to polepszyć to zrobić enuma, wtedy nie będzie problemu że ktoś np. literówkę zrobi

Reply:
Wystarczy wyciągnąć do osobnego pliku typu environment.ts jako funckję być może